docs/CHANGELOG.asciidoc

```diff
@@ -38,6 +38,33 @@ Review important information about the {kib} 8.4.x releases.
 
 Review the following information about the {kib} 8.4.0 release.
 
+[float]
+[[known-issue-8.4.0]]
+=== Known issues
+
+Do not upgrade Kibana from 8.3 to 8.4 if you have any alerting rules with a snooze notification configured.
```

Comment:
```suggestion
Do not upgrade Kibana from 8.3 to 8.4 if you have any alerting rules that are snoozed.
```

`snooze notification` seems confusing to me because snoozing will silence any notifications. Not sure if the above suggestion is the best tho

Reply:
👍 Thanks! I'll update.